builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -0,0 +1,41 @@
+local function clients_list_formspec(dialogdata)
+	local clients_list = server.clients_list
```

Comment:
Take it via the parameter:
```suggestion
	local clients_list = dialogdata.server.clients_list
```